ci_build/azure_pipelines/mobile-examples-pipeline.yml

```
@@ -280,3 +280,45 @@ jobs:
     parameters:
       xcWorkspacePath: 'mobile/examples/super_resolution/ios/ORTSuperResolution/ORTSuperResolution.xcworkspace'
       scheme: 'ORTSuperResolution'
+
+- job: QuestionAnsweringAndroid
+  pool:
+    vmImage: "macOS-11"
+
+  steps:
+  - template: templates/use-python-step.yml
+  - bash: |
+      set -e
+      bash ./prepare_model.sh
+    workingDirectory: 'mobile/examples/question_answering/android'
+    displayName: "Generate model"
+    
+  - script: |
+      python3 ./ci_build/python/run_android_emulator.py \
+        --android-sdk-root ${ANDROID_SDK_ROOT} \
+        --create-avd --system-image "system-images;android-30;google_apis;x86_64" \
```

"default" image seems to be stabler for CI tests
```suggestion
        --create-avd --system-image "system-images;android-30;default;x86_64" \
```